### PR TITLE
Share v2

### PR DESCRIPTION
**Pull request by octavioamu**
##### Checklist

- [ ] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Refers/Fixes

Refs: #3626
###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

**Comment by thelostone-mc** [comment]
Could we compress the svg via `svgo` ?

File: docs/STYLEGUIDE.md [hunk 1]
```diff
@@ -26,4 +28,23 @@ _usage_
...
+<textarea id="matchid">This text will be copied</textarea>
+<button data-copyclipboard="#matchid">Copy Text</button>
+```
+_usage with class_
+```
+
+<input type="text" class="matchclass" value="This text will be copied">
+<button data-copyclipboard=".matchclass">Copy Text</button>
+```
+_Note: You can use it with `textarea` or `input` elements._
```
Comment (thelostone-mc) [inline docs/STYLEGUIDE.md]: ❤️

File: docs/STYLEGUIDE.md [hunk 2]
```diff
@@ -26,4 +28,23 @@ _usage_
 The form styling used in Gitcoin can be found in
 [select.css](https://github.com/gitcoinco/web/blob/master/app/assets/v2/css/forms/select.css)
 
-_Note: All pages within gitcoin are expected to reuse these classes as applicable as opposed to declaring the `font` within the templates `css` file._
+### Copy to clipboard
```
Comment (SaptakS) [inline docs/STYLEGUIDE.md]: ❤️

File: app/dashboard/views.py [hunk 3]
```diff
@@ -747,6 +747,37 @@ def social_contribution_modal(request):
     params['promo_text'] = promo_text
     return TemplateResponse(request, 'social_contribution_modal.html', params)
 
+@csrf_exempt
```
Comment (owocki) [inline app/dashboard/views.py]: TODO - clean this up

**Pull request merged**